homeplug_exporter.go

```diff
@@ -131,6 +133,7 @@ func (n *HomeplugNetworkInfo) UnmarshalBinary(b []byte) error {
 			return err
 		}
 		n.Networks = append(n.Networks, ns)
+		log.Debugf("Network found: %s", &ns)
```

Comment:
This appears to use the old logrus logging (or standard library log package), and needs to be migrated to the go-kit/log API.

Reply:
argh, of course. This was old code and I just rebased it, sorry about that!